src/quo/components/text_input.cljs

```diff
@@ -148,7 +150,7 @@
     (fn [{:keys [label multiline error style input-style keyboard-type before after
                  cancel-label on-focus on-blur show-cancel accessibility-label
                  bottom-value secure-text-entry container-style get-ref on-cancel
-                 monospace auto-complete-type auto-correct]
+                 monospace auto-complete-type auto-correct new-ui?]
```

Comment:
@flexsurfer, @Parveshdhull is this how we are commonly dealing with adding the new UI? 
Seems like we could have a cleaner approach and we're just going to have a tonne of branches to clean up afterwards.

Reply:
it's a good question, actually, we should implement a new text input component, but for now, i guess we can use `new-ui?` , it's not cool, but i don't see a better solution for now

Reply:
okay, yeah I guess for now this is okay but we should give this some further consideration. This won't be the last migration in the codebase :)